src/yb/tools/yb-admin_client.cc

```diff
@@ -1738,10 +1738,33 @@ Status ClusterAdminClient::ModifyPlacementInfo(
   master::PlacementInfoPB* live_replicas = new master::PlacementInfoPB;
   live_replicas->set_num_replicas(replication_factor);
 
+  int total_min_replica_count = 0;
+
   // Iterate over the placement blocks of the placementInfo structure.
   std::unordered_map<std::string, int> placement_to_min_replicas;
-  for (int iter = 0; iter < placement_info_split.size(); iter++) {
-    placement_to_min_replicas[placement_info_split[iter]]++;
+  for (const auto& placement_block : placement_info_split) {
+    std::vector<std::string> placement_info_min_replica_split = strings::Split(
```

Comment:
Nit: Generally for complicated types we like to use `auto` to maintain readability. So, replace `std::vector<std::string>` with auto

Reply:
Tried doing that but it did not work. If it is auto, the type becomes Splitter<..> which does not have size method or indexing. Specifying vector does type conversion Splitter has a converter for Container types.
```
src/yb/tools/yb-admin_client.cc:1749:41: error: ‘class strings::internal::Splitter<strings::delimiter::Literal, strings::AllowEmpty>’ has no member named ‘size’
     if(placement_info_min_replica_split.size() == 0 ||
```
Also in repo, I see similar approach of specifying the type instead of auto so this will keep it consistent with other code as well. Let me know your thoughts.